radicle-cli/src/commands/remote/add.rs

```diff
@@ -0,0 +1,44 @@
+use radicle::node::tracking::store::Config;
+use radicle::{git::Url, node::TRACKING_DB_FILE, prelude::Id, Profile};
+use radicle_crypto::PublicKey;
+
+use crate::{git, terminal as term};
+
+/// core command to run the `rad remote add ...` subcommand.
+pub fn run(
+    repository: &git::Repository,
+    profile: &Profile,
+    pubkey: &PublicKey,
+    name: &Option<String>,
+    id: Id,
+) -> anyhow::Result<()> {
+    let name = if let Some(name) = name {
+        name.to_owned()
+    } else if let Some(name) = lookup_alias(profile, pubkey)? {
+        name
+    } else {
+        anyhow::bail!("an alias need to be specified");
+    };
```

Comment:
Putting the alias inside the function `lookup_alias` again, because it make the code a lot of cleaner.

cc @slack-coder